Wikipedia/Code/AboutViewController.m

```diff
@@ -270,8 +208,11 @@ - (BOOL)webView:(UIWebView*)webView shouldStartLoadWithRequest:(NSURLRequest*)re
     NSURL* requestURL = [request URL];
 
     if ([[self class] isLicenseURL:requestURL]) {
-        NSString* licenseText = [self licenseTextForLicenseURL:requestURL];
-        [self.webView loadHTMLString:licenseText baseURL:nil];
+        VTAcknowledgementsViewController* vc = [VTAcknowledgementsViewController acknowledgementsViewController];
+        vc.headerText = NSLocalizedString(@"We love open source software <3.", nil);
```

Comment:
I'm fine with just leaving it "We love open source software" (w/o the <3 part, which TWN may push back on)